acceptance_tests/features/inbox_internal.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Internal inbox
+  As an internal user
+  I need to view all inbox messages
+  So that I can work on the messages as required
+
+  Background: Internal user is already signed in
+    Given the internal user is already signed in
+
+  Scenario: If there are no messages the user will be informed of this.
```

Comment:
Can you add tags to these scenarios? So sm111_s01 etc?

Reply:
Added them